output/telegram.go

```diff
@@ -60,6 +65,10 @@ func (t *TelegramOutput) sendMessage(spanCtx sreCommon.TracerSpanContext, IDToke
 	span := t.tracer.StartChildSpan(spanCtx)
 	defer span.Finish()
 
+	if err := t.rateLimiter.Wait(context.TODO()); err != nil {
```

Comment:
Why not context.Background()? Or you plan to pass something more meaningful later?

Reply:
https://pkg.go.dev/context#Context - it can be spanCtx or context.TODO - but it is not  "main function, initialization, and tests, and as the top-level Context for incoming requests" (ok, afair spanCtx will be indirectly equal context.Background()